salt/salt.spec

```diff
@@ -791,10 +806,14 @@ install -Dd -m 0755 %{buildroot}%{_sysconfdir}/logrotate.d/
 install -Dpm 0644 salt/cli/support/profiles/* %{buildroot}%{python3_sitelib}/salt/cli/support/profiles
 
 # Install Salt tests
-install -Dd -m 0750 %{buildroot}%{_datadir}/salt
-install -Dd -m 0750 %{buildroot}%{_datadir}/salt/tests
-cp -a tests/* %{buildroot}%{_datadir}/salt/tests/
-sed -i '1s=^#!/usr/bin/\(python\|env python\)[0-9.]*=#!/usr/bin/python3=' %{buildroot}%{_datadir}/salt/tests/runtests.py
+install -Dd %{buildroot}%{python3_sitelib}/salt-testsuite
+cp -r -a tests %{buildroot}%{python3_sitelib}/salt-testsuite/
+# Remove runtests.py which is not used as deprecated method of running the tests
+rm %{buildroot}%{python3_sitelib}/salt-testsuite/tests/runtests.py
+# Copy conf files to the testsuite as they are used by the tests
+cp -r -a conf %{buildroot}%{python3_sitelib}/salt-testsuite/
+ln -s ../salt %{buildroot}%{python3_sitelib}/salt-testsuite/salt
+cp setup.py %{buildroot}%{python3_sitelib}/salt-testsuite/
```

Comment:
What is this needed for?

Reply:
setup.py was removed as decided not to run kitchen tests